concepts/observability-strategy/strategy.md

```diff
@@ -1,26 +1,25 @@
-# General Strategy
+# From Observability to Telemetry - a strategy shift in Kymas Observability feature
 
 ## Motivation
 
+The Kyma project evolved over time, from an extensibility framework having batteries-included, to a runtime which focusses on providing kubernetes building blocks with enterprise-grade quality. Hereby, the Kyma observability feature was not adapting yet, still providing an out-of-the box in-cluster stack which gets you started immediately, but does not qualify for the enterprise-grade qualities.
 
-In the current (2022) setup of Kyma, the observability stack provides an opinionated and lightweight solution out of the box, which solves basic requirements for application operators. It does not focus on integration aspects that would support, for example, cross-runtime observations, advanced analytic tools, or reuse of users' existing observability infrastructure. Also, the current observability stack does not provide a guide how to extend the setup to become highly available (HA) with historical storage of the data.
+Observability backends are available in various ways from different vendors as managed services. While users can pick and choose and can leverage benefits like cross-cluster scenarios, the integration aspect into that external managed services is often complex and heterogenous. Kyma should not invest into another opinionated solution for an in-cluster backend with enterprise-grade qualities, it should instead focus on simplifying the instrumentation and collection of the telemetry data and the integration of the data into external solutions. Hereby, it will leverage as much as possible the OpenTelemetry standards to provide integration options in a vendor-neutral approach.
 
-With such limited integration possibilities and providing only a very lightweight in-cluster solution, the current observability stack is missing a lot of usage scenarios, and the potential audience is very narrow. 
-By moving the focus away from backend solutions towards vendor-neutral integration possibilities, Kyma runtime will meet the needs of a broader audience and open up possibilities for wider adoption.
+With that Kyma will shift the focus away from providing a full Observability solution, instead it will simplify the instrumentation and integration of telemetry data into external systems.
```

Comment:
```suggestion
With that, Kyma will shift the focus away from providing a full Observability solution; instead it will simplify the instrumentation and integration of telemetry data into external systems.
```